### ISSUE TITLE
Regarding Training Effectiveness

### ISSUE THREAD

**Issue by Azer90** [clovaai/fewshot-font-generation#23]
Hello, I would like to ask if the effect shown in the image below is normal when I reached 550000 steps in the first stage of training using LFfont. Is it because the training was not done properly or is there a way to modify hyperparameters? Or is it not possible

**Comment by Azer90** [comment]
![Image](https://github.com/user-attachments/assets/01212491-2036-4f96-9f54-dd52a5d618a7)

**Comment by Azer90** [comment]
![Image](https://github.com/user-attachments/assets/b51b2976-08c3-4b90-98b5-f2076f057d8f)

**Comment by Azer90** [comment]
![Image](https://github.com/user-attachments/assets/2625e09c-f0d9-410d-96d4-2be7e307433b)

**Comment by Azer90** [comment]
Could you please help me check if the loss information for this training is normal? Thank you
